src/OpenFin/openfin.ts

```diff
@@ -142,7 +142,8 @@ export class OpenFinContainer extends WebContainerBase {
         height: { target: "defaultHeight" },
         width: { target: "defaultWidth" },
         taskbar: { target: "showTaskbarIcon" },
-        center: { target: "defaultCentered" }
+        center: { target: "defaultCentered" },
+        show: { target: "autoShow" }
```

Comment:
The option for OpenFin is "autoShow" so map the desktopJS option of "show" to "autoShow".